ui/components/NavigatorExtension.js

```diff
@@ -139,6 +146,14 @@ function NavigatorExtension({
             useFilterK8sContexts,
             useDynamicComponent,
           },
+          store: store,
+          currentOrg: setOrganization,
+          reduxHooks: {
```

Comment:
dont pass the whole store , as it would add additional dependencies on meshery . only pass a object `mesheryStore` which would include setters , getters and selectors for the exposed properties  ( in this case oganization )

Reply:
eg mesheryStore  : { organization : { set : () => dispatch(....) , get : ()=> store.get(...) , useCurrentOrg : () => useSelector(useCurrentOrgSelector)) }